lib/pure/os.nim

```diff
@@ -935,6 +933,43 @@ proc getConfigDir*(): string {.rtl, extern: "nos$1",
     result = getEnv("XDG_CONFIG_HOME", getEnv("HOME") / ".config")
   result.normalizePathEnd(trailingSep = defined(nimLegacyHomeDir))
 
+proc getCacheDir*(): string =
+  ## Returns the cache directory of the current user for applications.
+  ## 
+  ## on windows: `getEnv("LOCALAPPDATA")`
+  ##
+  ## on osx: `getEnv("XDG_CACHE_HOME", getEnv("HOME") / "Library/Caches")`
+  ##
+  ## else: `getEnv("XDG_CACHE_HOME", getEnv("HOME") / ".cache")`
+  ##
+  ## See also:
+  ## * `getHomeDir proc <#getHomeDir>`_
+  ## * `getTempDir proc <#getTempDir>`_
+  ## * `getConfigDir proc <#getConfigDir>`_
+  # follows https://crates.io/crates/platform-dirs
+  runnableExamples:
+    from std/strutils import endsWith
+    assert not getCacheDir().endsWith DirSep
```

Comment:
What's the point of this example? This looks like it should be in a separate test file, if anything.

Reply:
Yes, please stop abusing runnable examples for tests that shouldn't be in the docs

Reply:
done